package.json

```diff
@@ -3,11 +3,12 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-    "@material-ui/core": "latest",
+    "@material-ui/core": "^4.11.0",
```

Comment:
I think we should leave them at "latest" for now, we can move to select a version for them all once we are ready, but I would like to avoid mixing the two.